packages/extension/src/permission/FirefoxPermissionDeclined.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { ReactElement } from 'react';
+import {
+  cloudinaryFeedBgLaptop,
+  cloudinaryGenericErrorDark,
+} from '@dailydotdev/shared/src/lib/image';
+import {
+  Typography,
+  TypographyColor,
+  TypographyTag,
+  TypographyType,
+} from '@dailydotdev/shared/src/components/typography/Typography';
+import { FirefoxPermissionContainer } from './common';
+
+interface FirefoxPermissionDeclinedProps {
+  onGoBack(): void;
+}
+
+export function FirefoxPermissionDeclined({
+  onGoBack,
+}: FirefoxPermissionDeclinedProps): ReactElement {
+  return (
+    <FirefoxPermissionContainer className="justify-center">
+      <img
+        src={cloudinaryFeedBgLaptop}
+        alt="a glowing background"
+        className="pointer-events-none absolute -top-10 -z-1 select-none"
+      />
+      <div className="flex w-full max-w-[35rem] flex-col items-center gap-4">
+        <span className="h-36 max-w-[16.25rem]">
+          <img src={cloudinaryGenericErrorDark} alt="You declined (FML)" />
+        </span>
+        <Typography
+          tag={TypographyTag.H1}
+          type={TypographyType.LargeTitle}
+          bold
+        >
+          You declined.
+        </Typography>
+        <Typography
+          color={TypographyColor.Secondary}
+          tag={TypographyTag.H1}
+          type={TypographyType.Body}
+          className="text-center"
+        >
+          We understand your choice, but just so you know, we’ll need those
+          permissions to deliver any content. Without them, it’s like trying to
+          code without a keyboard!
+        </Typography>
+        <Typography
+          color={TypographyColor.Secondary}
+          tag={TypographyTag.H1}
+          type={TypographyType.Body}
+        >
+          You can{' '}
+          <button type="button" className="text-text-link" onClick={onGoBack}>
+            go back
```

Comment:
Since the user has already declined, maybe we should also show a link to the webapp as an alternative?

Reply:
Good point. I can also raise it to the product team 👀

Reply:
Added! 🚀